### PR TITLE
Add missing system checks for custom storage directories

### LINKED ISSUE #52: Chore missing system-check
Add missing system-check for COMPANY_EMPLOYEES_STORAGE setting

### PR DESCRIPTION
**Pull request by maciejSamerdak**
Resolves #52 

Adds missing system checks for custom storage directories.

### CODE REVIEW

File: project_liberation/system_check.py [hunk 1]
```diff
@@ -26,3 +33,23 @@ def check_google_api_key(app_configs: Any, **kwargs: Any) -> list:  # pylint: di
         return [GOOGLE_API_KEYS_ERRORS["length"]]
 
     return []
+
+
+def check_custom_storage_directories(setting_name: str) -> list:
+    if not hasattr(settings, setting_name):
+        return [setting_not_declared_error(setting_name)]
+
+    if not isinstance(settings.__getattr__(setting_name), str):
```
Comment (kbeker) [inline project_liberation/system_check.py]: Please change it into `getattr` function

File: project_liberation/system_check.py [hunk 2]
```diff
@@ -26,3 +33,23 @@ def check_google_api_key(app_configs: Any, **kwargs: Any) -> list:  # pylint: di
...
+
+
+@register()
+def check_company_employees_storage(app_configs: Any, **kwargs: Any) -> list:  # pylint: disable=unused-argument
+    return check_custom_storage_directories("COMPANY_EMPLOYEES_STORAGE")
+
+
+@register()
+def check_testimonial_photos_storage(app_configs: Any, **kwargs: Any) -> list:  # pylint: disable=unused-argument
+    return check_custom_storage_directories("TESTIMONIAL_PHOTOS_STORAGE")
```
Comment (kbeker) [inline project_liberation/system_check.py]: You can create from these two system checks one with map to settings names which has to be checked.

Comment (maciejSamerdak) [reply project_liberation/system_check.py]: Done

File: project_liberation/system_check.py (hunk 1 above)
Comment (maciejSamerdak) [reply project_liberation/system_check.py]: Done

**Pull request merged**